packages/shared-data/plans.ts

```diff
@@ -48,11 +51,27 @@ export const plans: PricingInformation[] = [
     name: 'Pro',
     nameBadge: '',
     costUnit: '/ month / project',
+    costUnitOrg: '/ month',
     href: 'https://supabase.com/dashboard/new/new-project',
     priceLabel: 'From',
     warning: 'Usage-based plan',
     priceMonthly: 25,
     description: 'For production applications with the option to scale.',
+    featuresOrg: [
+      '$10 Compute Credits',
```

Comment:
should we keep the casing consistent? same for the other lines below too ("Realtime" vs "realtime" etc)
```suggestion
      '$10 compute credits',
```